podcasts/PodcastImage.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+import Kingfisher
+import PocketCastsServer
+
+struct PodcastImage: View {
+    let uuid: String
+    let size: Int
+
+    init(uuid: String, size: Int = 280) {
+        self.uuid = uuid
+        self.size = size
+    }
+
+    var body: some View {
+        KFImage(ServerHelper.imageUrl(podcastUuid: uuid, size: size))
```

Comment:
If we use the podcast image size enum we can use the ImageManager to get the url:
```suggestion
        KFImage(ImageManager.sharedManager.podcastUrl(imageSize: size, uuid: uuid))
```

Reply:
Done in c3adc37cfc97454a4815fa92f1404c52678a2f5a